Modules/grpmodule.c

```diff
@@ -124,11 +124,11 @@ grp_getgrgid_impl(PyObject *module, PyObject *id)
         Py_DECREF(py_int_id);
     }
 #ifdef HAVE_GETGRGID_R
-    Py_BEGIN_ALLOW_THREADS
     int status;
     Py_ssize_t bufsize;
     struct group grp;
```

Comment:
Could you please add a comment about `grp` is used via the `p` pointer.

Reply:
I've added comments.